### PR TITLE
Imports - more details

### PR DESCRIPTION
**Pull request by SomMeri**
I added some more details about less file imports:
- what happens when the import statement contains media queries,
- what happens when it is placed into ruleset or mixin,
- import-multiple statement.

### CODE REVIEW

File: templates/pages/doc.md [hunk 1]
```diff
@@ -562,25 +562,97 @@ they don't show up in the compiled CSS output:
...
 
-You can import `.less` files, and all the variables and mixins in them will be made available to the main file.
-The `.less` extension is optional, so both of these are valid:
+You can import both css and less files. Only less files import statements are processed, css file import statements are kept as they are. If you want to import a CSS file, and don't want LESS to process it, just use the `.css` extension:
+
+    @import "lib.css";
+
+Compilation makes only one change to css file imports: top level css file imports are moved on top of the sheet, right after @charset declarations. 
+
+<table class="code-example" cellpadding="0">
```
Comment (lukeapage) [inline templates/pages/doc.md]: this is mark-down, not html, so just indent less code by 4 spaces

**Comment by lukeapage** [comment]
thanks, great help

File: templates/pages/doc.md (hunk 1 above)
Comment (SomMeri) [inline templates/pages/doc.md]: This was supposed to show two codes side by side - less on the left and css on the right - I through that it is more readable.  But, if you do not like it I can move them to usual places - less first and css output below that.

**Comment by lukeapage** [comment]
pulled, sorry for the delay

**Pull request closed without merging**